annotations/static/annotations/js/annotators/appellationlist.js

```diff
@@ -217,6 +217,7 @@ AppellationList = {
             // Replace an array prop wholesale doesn't seem to trigger a
             //  DOM update in the v-for binding, but a push() does; so we'll
             //  just push the appellations that aren't already in the array.
+            this.current_appellations = [];
```

Comment:
I'm confused, wouldn't this make the next line useless? Doesn't the next line create a list of ids of current appellations, which now would always be an empty list?

Reply:
That's right. we don't need those lines.  Just `this.current_appellations = this.appellations;` will work. 
Can you confirm if the functionality of the buttons mentioned below is correct?
<img width="853" alt="Screenshot 2024-12-06 at 10 06 22 AM" src="https://github.com/user-attachments/assets/9bde3ef6-0e9e-4c11-8101-bb3cb41c1703">